devtools/server/actors/replay/module.js

```diff
@@ -1048,6 +1059,200 @@ function Debugger_getSourceContents({ sourceId }) {
   };
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// Network Commands
+///////////////////////////////////////////////////////////////////////////////
+
+let gCurrentRequestEvent;
+
+if (isRecordingOrReplaying) {
+  const gPendingRedirects = new Map();
+  const gActiveRequests = new Map();
```

Comment:
These globals need comments.  This also looks like a good place to add an overview comment explaining how network commands are intercepted and conveyed to the record/replay driver.